docs/howtoguides/configure_proxies.rst

```diff
@@ -10,6 +10,12 @@ addition, the Ubuntu Pro Client will automatically set up proxies for all
 programs required for enabling Ubuntu Pro services. This includes APT, snaps,
 and Livepatch.
 
+.. note::
+
+    Throught this guide, we will set the https proxy with an http proxy.
```

Comment:
Not sure if this is what you meant but the original sentence felt weird
```suggestion
    Throught this guide, we will set the https proxy with an http prefix.
```

Reply:
Good catch @a-dubs. Updating it now